src/test/kotlin/com/terraformation/backend/species/db/SpeciesStoreTest.kt

```diff
@@ -363,4 +364,23 @@ internal class SpeciesStoreTest : DatabaseTest(), RunsAsUser {
 
     assertThrows<OrganizationNotFoundException> { store.findAllSpecies(organizationId) }
   }
+
+  @Test
+  fun `acceptProblemSuggestion throws ScientificNameExistsException on name collision`() {
+    store.createSpecies(
+        SpeciesRow(organizationId = organizationId, scientificName = "Species name"))
+    val badSpeciesId =
+        store.createSpecies(
+            SpeciesRow(organizationId = organizationId, scientificName = "Bad name"))
```

Comment:
little confused here. Does this throw because there already exists a species with the scientific name "Bad name"?

Reply:
No, because there's already one with the suggested name ("Species name"). I'll update this test to make that clearer.

Reply:
Ohh, i see the flow of things now